settings.gradle

```diff
@@ -6,6 +6,7 @@ include ':third_party:gif_encoder'
 include ':samples:flickr'
 include ':samples:giphy'
 include ':samples:svg'
+include ':samples:contacturi'
```

Comment:
I think if we're introducing a new project, it may worth doing a catch-all one, so there's a place where we can put single-activity manual tests (I would go as far as merging the other samples into one big sample project in the future, of course it's out of scope for this PR), so instead of `contactsuri` it would be better to call this `misc`/`tests`/`manualTest` or similar.